pages/index.js

```diff
@@ -1,3 +1,17 @@
+import Landingpage from "@/componentens/Landingpage";
+import styled from "styled-components";
 export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+  return (
+    <>
+      {" "}
+      <Title>Your Travel Journal App</Title>
+      <Landingpage />
```

Comment:
I recommend not turning the Landingpage into a component and instead copying all content from the component into this page component